### ISSUE TITLE
Numbers which ends 5s are sorted like µs

### ISSUE THREAD

**Issue by pichi** [timbunce/devel-nytprof#121]
All numbers in seconds which ends 5 are sorted like it is actually in microseconds. Tested in Chrome (Version 66.0.3359.117 (Official Build) beta (64-bit)) and Firefox (59.0.2 (64-bit)).

There is a fix #120 ready for merge.

**Comment by pichi** [comment]
BTW, there is how to fix existing reports:

```bash
grep -rlF '/[µ\\xB5]s/' . | xargs -I '{}' sed -i 's./\[µ\\\\xB5\]s/./[µ\\xB5]s/.' '{}'
```

**Comment by timbunce** [comment]
Closed by #120.
